DOCS/OSI-SPDX-history.md

```diff
@@ -0,0 +1,104 @@
+And add the SPDX id, `Artistic-1.0-Perl` on [this page](https://opensource.org/licenses/Artistic-Perl-1.0) and update that note accordingly as well. 
+
+## License exceptions
+In at least one case, the OSI approved a license exception, referred to as [eCos License version 2.0](https://opensource.org/licenses/eCos-2.0) and lists the SPDX identifiers as "eCos-2.0". This is probably because this license was originally on the SPDX License List as a stand alone license. However, when SPDX 2.0 came out in May 2015, exceptions were moved to their own part of the SPDX License List to be used with the `WITH` operator to allow more extensible and varied license expressions, see https://spdx.org/licenses/eCos-2.0.html and now see https://spdx.org/licenses/eCos-exception-2.0.html
+
+Note that OSI has the GPL-2.0-or-later license notice on its page for the eCos exception, which would use the SPDX license expression: `GPL-2.0-or-later WITH eCos-exception-2.0`. Did OSI _only_ approve this exception for GPL-2.0-or-later? 
+
+Suggestion: Update the [eCos](https://opensource.org/licenses/eCos-2.0) page to use the current `eCos-exception-2.0` identifier and change the URL to https://opensource.org/licenses/eCos-exception-2.0. Clarify what versions of GPL it is approved for.
+
+## Licenses that allow for variations by way of a different license notice
+These are licesnes that were approved by the OSI in full. These licenses allow legally substantive variations based on use of a different license notice. This variataion warranted distinct SPDX identifiers to indicate whether or not the allowed variation is being triggered. All of these variations include a Note in the SPDX License List entry explaining this. 
+
+* `CAL-1.0-Combined-Work-Exception`
+* `MPL-2.0-no-copyleft-exception`
+* `OFL-1.1-no-RFN`
+* `OFL-1.1-RFN`
+
+Suggestion: OSI could add the variant SPDX ids to the license pages.
+
+## Not in SPDX list, but in OSI's list
+
+* [Jabber Open Source License](https://opensource.org/licenses/jabberpl) - This is a case where the text on the OSI site is not the same as the archived text at http://archive.jabber.org/core/JOSL.pdf SPDX would consider these two texts as different licenses, and was trying to clarify with the OSI which text was submitted and approved by OSI. Martin Michlmayr and Jilayne discussed this back in 2013 (https://lists.spdx.org/g/Spdx-legal/topic/22080295#475), but never reached a resolution. Jilayne emailed the OSI Board about this issue in July 2019. 
+
+Suggestion: SPDX could recognize both variants on the SPDX License List and note the one that is specifically OSI-approved is the one that matches that exact text on the OSI site?  This would require distinct names and identifiers. 
+
+* CVW: MITRE Collaborative Virtual Workspace License
+
+still need to research issue here
+
+# from Max's list - still to address: 
+## Not in OSI list, but in SPDX's list
+
+* GPL-3.0-with-GCC-exception
+
+what is the quesiton here?
```

Comment:
@richardfontana explained the background of the WITH IDs:

> You list some cases of 'WITH' expressions. The OSI has been reluctant to approve license exceptions, except in a few special cases where the exception (or exception coupled with a standard license) is itself thought of as a single license (e.g. LGPL version 3; ec0s-2.0 is also like this). From my recollection of my time on the OSI board, the main concern was the potential numerosity of license submissions if the OSI encouraged submission of exceptions. There's been a tendency to assume that typical types of GPL exceptions are legit (for a GPL-world notion of legit) because they conform to the model of a grant of additional permission -- I need to comment on this issue on another recent thread.

That would resolve my issue with this ID as I can assume that an official SPDX expression containing `with` is very likely OSI-approved.

Reply:
I don't think "OSI-approved" is the correct concept here. Rather, historically, the OSI might have informally said that "normal" (additional-permission-style) GPL exceptions are _probably_ OSD-conformant. There's no formal sense in which OSI has approved anything other than the licenses it has approved through its license approval process.

Also, I don't think you can make this general assumption going forward, because in a recent mailing list thread @jlovejoy has [clarified](https://lists.spdx.org/g/Spdx-legal/message/3283) (unless I misunderstood) that SPDX is not, or no longer, strictly tied to an understanding that "exceptions" (the right side of a WITH expression) must be limited to additional permissions. This came out of some interest in exploring SPDX recognition of a currently-hypothetical exception type that would involve imposition of clearly-non-FOSS restrictions on top of FOSS license terms. In the past, SPDX has [rejected](https://github.com/spdx/license-list-XML/issues/902) inclusion of an exception identifier for the Commons Clause but I believe this was prior to SPDX's adoption of its current license incluson criteria.

Reply:
seems like there is no longer a question here, so removing this from the "list"